unfetter-discover/unfetter-ui
# Added subscriber for current capability to update the title on the he…

Fixes unfetter-discover/unfetter#1017

to test:
go to baselines app
create a new baseline
Add title, continue, add categories, continue, add capabilities, continue
Click a capability on the left (in the stepper) and see that the title is updated.
